### ISSUE TITLE
Minor incompatibility with Bootstrap Accessibility Plugin

### ISSUE THREAD

**Issue by IngwiePhoenix** [mythz/jquip#60]
The accessibility plugin for Bootstrap, developed by eBay/PayPal seems to be incompatible with jquip. The respective lines of failing code:

`bootstrapaccessibilityplugin/src/js/functions.js`:
```javascript
      $.extend( $.expr[ ":" ], {
```

`bootstrapaccessibilityplugin/src/js/alert.js`:
```javascript
    $('.close').removeAttr('aria-hidden').wrapInner('<span aria-hidden="true"></span>').append('<span class="sr-only">Close</span>')
```

`bootstrap/src/js/modal.js`:
This only happens when you trigger a modal. So far the first incompatible bootstrap part.
```javascript
        var e    = $.Event('show.bs.modal', { relatedTarget: _relatedTarget })
```

`bootstrap/src/js/dropdown.js`:
This error happens when you trigger a dropdown.
```javascript
$parent.trigger(e = $.Event('show.bs.dropdown', relatedTarget))
```

Hopefully these could be resolved! :)
